Sources/App/Features/Past Sponsors/Transformers/PastSponsorTransformer.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import Vapor
+
+enum PastSponsorTransformer: Transformer {
+    static func transform(_ entity: PastSponsor?) -> PastSponsorResponse? {
+        guard let entity = entity else {
+            return nil
+        }
+        
+        guard let bucketName = Environment.get("S3_BUCKET_NAME") else {
+            fatalError("Missing 'S3_BUCKET_NAME' environment variable")
+        }
+        
+        return .init(
+            id: entity.id,
+            name: entity.name,
+            image: "https://\(bucketName).s3.eu-west-2.amazonaws.com/\(entity.image)",
```

Comment:
We could probably go back through and cleanup old examples, but we don't need to do this AWS image string interpolation any more.

We just need to return `entity.image` and then within the .leaf file itself use `#awsImage(image)`.

The awsImage tag will do the interpolation for you, aiding in re-use and simplifying the transformers.